app/forms/questionnaire_form.py

```diff
@@ -70,6 +71,26 @@ def map_errors(self):
     def answer_errors(self, input_id):
         return [error[1] for error in self.map_errors() if input_id == error[0]]
 
+    def get_data(self, answer_id):
+        attr = getattr(self, answer_id)
+        return attr.raw_data[0] if attr.raw_data else ''
+
+    def option_has_other(self, answer_id, option_index):
+        if not self.options_with_children:
+            self.options_with_children = SchemaHelper.get_parent_options_for_block(self.block_json)
```

Comment:
Nope, but we will as part of the request for the block, which this call will cache.